register.html

```diff
@@ -20,15 +20,47 @@ <h1 class="logo">Relaxr</h1>
 					<ul>
 						<li><a href="index.html">Home</a></li>
 						<li><a href="blog.html">Blog</a></li>
-						<li><a href="#">Contact</a></li>
+						<li><a href="contact.html">Contact</a></li>
```

Comment:
since you called the file `register.html`, this `href` should equal `"register.html"`